src/Microsoft.TemplateEngine.Cli/LocalizableStrings.resx

```diff
@@ -793,6 +793,10 @@ The header is followed by the list of parameters and their errors (might be seve
     <value>To use built-in template package, uninstall manually installed template package using:</value>
     <comment>followed by command to uninstall package</comment>
   </data>
+  <data name="TemplateListCoordinator_Error_FailedConstraints" xml:space="preserve">
+    <value>{0} template(s) are not displayed due to their constraints are not satisfied. To display them, use '{1}' option.</value>
```

Comment:
I think show/hide are the verbs we should use when talking about data that is or is not shown. Slightly more concise and it echoes the language of the CLI library (shared concepts are good)

```suggestion
    <value>{0} template(s) are hidden because their constraints are not satisfied. To show them, use the '{1}' option.</value>
```